cmd/lotus-worker/tasks.go

```diff
@@ -45,32 +45,52 @@ var tasksEnableCmd = &cli.Command{
 	Name:      "enable",
 	Usage:     "Enable a task type",
 	ArgsUsage: "[" + settableStr + "]",
-	Action:    taskAction(api.Worker.TaskEnable),
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "all",
+			Usage: "Enable all task types",
+		},
+	},
+	Action: taskAction(api.Worker.TaskEnable),
 }
 
 var tasksDisableCmd = &cli.Command{
 	Name:      "disable",
 	Usage:     "Disable a task type",
 	ArgsUsage: "[" + settableStr + "]",
-	Action:    taskAction(api.Worker.TaskDisable),
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:  "all",
+			Usage: "Disable all task types",
+		},
+	},
+	Action: taskAction(api.Worker.TaskDisable),
 }
 
 func taskAction(tf func(a api.Worker, ctx context.Context, tt sealtasks.TaskType) error) func(cctx *cli.Context) error {
 	return func(cctx *cli.Context) error {
-		if cctx.NArg() != 1 {
-			return xerrors.Errorf("expected 1 argument")
+		allFlag := cctx.Bool("all")
+
+		if cctx.NArg() == 1 && allFlag {
+			return xerrors.Errorf("Cannot use --all flag with task type argument")
+		}
+
+		if cctx.NArg() == 0 && !allFlag {
+			return xerrors.Errorf("Expected 1 argument or use --all flag")
 		}
```

Comment:
I'd do
```suggestion
		if cctx.NArg() != 1 && !allFlag {
			return xerrors.Errorf("Expected 1 argument or use --all flag")
		}
```

Otherwise it's possible to call this with e.g. two args and no --all, and the command would just not do anything